src/components/OrgPostCard/OrgPostCard.tsx

```diff
@@ -15,39 +19,93 @@ interface OrgPostCardProps {
 }
 
 function OrgPostCard(props: OrgPostCardProps): JSX.Element {
+  const [postformState, setPostFormState] = useState({
+    posttitle: '',
+    postinfo: '',
+  });
+
+  useEffect(() => {
+    setPostFormState({
+      posttitle: props.postTitle,
+      postinfo: props.postInfo,
+    });
+  }, []);
+
   const [create] = useMutation(DELETE_POST_MUTATION);
+  const [updatePost] = useMutation(UPDATE_POST_MUTATION);
 
   const DeletePost = async () => {
-    const sure = window.confirm('Are you sure you want to delete Post ?');
-    if (sure) {
-      try {
-        const { data } = await create({
-          variables: {
-            id: props.id,
-          },
-        });
+    try {
+      const { data } = await create({
+        variables: {
+          id: props.id,
+        },
+      });
 
-        /* istanbul ignore next */
+      /* istanbul ignore next */
+      if (data) {
+        window.alert('Post deleted successfully.');
         window.location.reload();
-      } catch (error) {
-        window.alert(error);
       }
+    } catch (error: any) {
+      /* istanbul ignore next */
+      toast.error(error.message);
+    }
+  };
+
+  const handleInputEvent = (e: ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+
+    setPostFormState({ ...postformState, [name]: value });
+  };
+
+  const updatePostHandler = async (e: ChangeEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    try {
+      const { data } = await updatePost({
+        variables: {
+          id: props.id,
+          title: postformState.posttitle,
+          text: postformState.postinfo,
+        },
+      });
+
+      /* istanbul ignore next */
+      if (data) {
+        window.alert('Post Updated successfully.');
```

Comment:
dont user alert here it doesnt look good

Reply:
JavaScript alert has been replaced with react-toast alert in the last commit.